src/components/MusicPage/MusicPage.js

```diff
@@ -9,53 +9,79 @@ import {
   ShowFormatCooldown,
 } from '../WorkoutCard/ShowFormatFromSections'
 import { selectColor, selectColorByInterval } from '../../services/selectColor'
+import Header from '../Header'
 
 export default function MusicPage({ location, onCreatePlaylist }) {
   const { name, warmup, intervalsT, cooldown } = location.workout
 
   return (
-    <Grid>
-      <Heading> Select your music:</Heading>
-
-      <MusicForm
-        workout={location.workout}
-        onCreatePlaylist={onCreatePlaylist}
-      />
-
-      <CardGrid>
-        <Title>
-          Your selected workout:
-          <br /> {name}
-        </Title>
-
-        <Wrapper>
-          <div>
-            <WarmupSection background={selectColor(warmup.powerHigh)}>
-              <ShowFormatWarmup warmup={warmup} />
-            </WarmupSection>
-            <IntervalsTSection
-              background={selectColorByInterval(intervalsT.onPower)}
-            >
-              <ShowFormatIntervalsT intervalsT={intervalsT} />
-            </IntervalsTSection>
-            <CooldownSection background={selectColor(cooldown.powerHigh)}>
-              <ShowFormatCooldown cooldown={cooldown} />
-            </CooldownSection>
-          </div>
-        </Wrapper>
-      </CardGrid>
-
-      <BackButton as={NavLink} to={{ pathname: '/' }}>
-        <Icon glyph="back" size={24} />
-        Back
-      </BackButton>
-    </Grid>
+    <div>
+      <HeaderStyled>Music</HeaderStyled>
+
+      <Grid>
+        <Heading> Select your music:</Heading>
+
+        <MusicForm
+          workout={location.workout}
+          onCreatePlaylist={onCreatePlaylist}
+        />
+
+        <CardGrid>
+          <Title>
+            Your selected workout:
+            <br /> {name}
+          </Title>
+
+          <Wrapper>
+            <div>
+              <WarmupSection background={selectColor(warmup.powerHigh)}>
+                <ShowFormatWarmup warmup={warmup} />
+              </WarmupSection>
+              <IntervalsTSection
+                background={selectColorByInterval(intervalsT.onPower)}
+              >
+                <ShowFormatIntervalsT intervalsT={intervalsT} />
+              </IntervalsTSection>
+              <CooldownSection background={selectColor(cooldown.powerHigh)}>
+                <ShowFormatCooldown cooldown={cooldown} />
+              </CooldownSection>
+            </div>
+          </Wrapper>
+        </CardGrid>
+
+        <BackButton as={NavLink} to={{ pathname: '/' }}>
+          <Icon glyph="back" size={24} />
+          Back
+        </BackButton>
+      </Grid>
+    </div>
   )
 }
 
+const HeaderStyled = styled(Header)`
+  position: fixed;
+  top: 0;
+  left: 0;
+`
 const Grid = styled.section`
   display: grid;
   gap: 20px;
+  overflow-y: scroll;
+  &::before {
+    content: '';
+    display: block;
+    height: 58px;
+    width: 100%;
+  }
+  &::after {
+    content: '';
+    display: block;
+    height: 58px;
+    width: 100%;
+  }
```

Comment:
why the before and after they have the same style?

Reply:
this is more less my grid. whatever in between plus 2 sections over and under with the same height. I will consider this for the final layout.